Formula/g/github-mcp-server.rb

```diff
@@ -0,0 +1,18 @@
+class GithubMcpServer < Formula
+  desc "GitHub Model Context Protocol server for AI tools"
+  homepage "https://github.com/github/github-mcp-server"
+  url "https://github.com/github/github-mcp-server/archive/refs/tags/v0.1.0.tar.gz"
+  sha256 "c390bcf344802f8ac81c65e5fcb72b9c47dba4090cec1bb08698311e334d399b"
+  license "MIT"
+
+  depends_on "go" => :build
+
+  def install
+    system "go", "build", *std_go_args, "./cmd/github-mcp-server"
+  end
+
+  test do
+    output = shell_output("#{bin}/github-mcp-server --help")
+    assert_match "A GitHub MCP server that handles various tools and resources.", output
+  end
```

Comment:
```suggestion
  test do
    assert_match version.to_s, shell_output("#{bin}/github-mcp-server --version")

    ENV["GITHUB_PERSONAL_ACCESS_TOKEN"] = "test"

    json = <<~JSON
      {
        "jsonrpc": "2.0",
        "id": 3,
        "params": {
          "name": "get_me"
        },
        "method": "tools/call"
      }
    JSON

    assert_match "GitHub MCP Server running on stdio", pipe_output(bin/"github-mcp-server stdio 2>&1", json, 0)
  end
```

Reply:
@MayorFaj @chenrui333 can you explain what the purpose of passing:

```
ENV["GITHUB_PERSONAL_ACCESS_TOKEN"] = "test"
    json = <<~JSON
      {
        "jsonrpc": "2.0",
        "id": 3,
        "params": {
          "name": "get_me"
        },
        "method": "tools/call"
      }
    JSON
```

Is? It seems like it would just return an error? Maybe we should be using `initialize`?

https://github.com/modelcontextprotocol/modelcontextprotocol/blob/main/schema/2025-03-26/schema.ts#L164-L174

Reply:
In fact, this looks like it will result on an unnecessary call to the GitHub API, providing no value but additional load.